.github/workflows/backport.yml

```diff
@@ -0,0 +1,22 @@
+---
+name: Backport Assistant Runner
+
+on:
+  pull_request:
+    types:
+      - closed
+      - labeled
+
+jobs:
+  backport:
+    if: github.event.pull_request.merged
+    runs-on: ubuntu-latest
+    container: hashicorpdev/backport-assistant:0.2.2
+    steps:
+      - name: Run Backport Assistant
+        run: |
+          backport-assistant backport
+        env:
+          BACKPORT_LABEL_REGEXP: "(?P<target>\\d+\\.\\d+\\.x)-backport"
+          BACKPORT_TARGET_TEMPLATE: "release/{{.target}}"
```

Comment:
Consul's version of this is slightly different. Should we be doing the same?

https://github.com/hashicorp/consul/blob/892032acae4b2f93e2523afe89d3725310d3755b/.github/workflows/backport-assistant.yml#L52

Reply:
Ah I'm seeing in the [docs](https://github.com/hashicorp/backport-assistant) that `target` just needs to match what's parsed out of the label. Since we're also parsing the `.x` in our capture group above, we don't need to add the `.x` as Consul does here.

Reply:
I basically just copied over the example in the docs, open to doing it the way consul does